packages/one-app-bundler/README.md

```diff
@@ -212,6 +213,22 @@ negative impact on performance_.**
 }
 ```
 
+#### Specify what version of One App your module is compatible with
+
+You can specify which version of One App you module is compatible with by simply adding the below configuration to your `package.json`.
+
+```json
+{
+  "one-amex": {
+    "bundler": {
+      "app": {
```

Comment:
i think this is just `appCompatibility` 
https://github.com/americanexpress/one-app-cli/blob/696609c702b128ba0339064173ac328ce8c00766/packages/one-app-bundler/webpack/loaders/validate-one-app-compatibility-loader.js#L18

Reply:
https://github.com/americanexpress/one-app-cli/blob/c5a3e82d1c4e778cc05b24734390f938d7f984b6/packages/one-app-bundler/utils/getConfigOptions.js#L53